Dockerfile

```diff
@@ -2,8 +2,11 @@ ARG BUILD_FROM
 FROM ${BUILD_FROM}
 
 # Synchronize with homeassistant/core.py:async_stop
+# Add Home Assistant wheels index for newer versions
+
 ENV \
-    S6_SERVICES_GRACETIME=220000
+    S6_SERVICES_GRACETIME=220000 \
+    EXTRA_INDEX_URL=https://wheels.home-assistant.io/musllinux-index/
```

Comment:
for testing only, remove this in favor of https://github.com/home-assistant/docker/pull/297